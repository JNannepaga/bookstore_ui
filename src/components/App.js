import './App.css';

function App() {
  return (
    <div>
      <h1>Welcome to bookstore App.</h1>
    </div>
  );
}

export default App;
