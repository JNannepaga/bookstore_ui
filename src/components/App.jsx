import React from 'react';
import './App.css';
import {BookManager} from "./BookManager";

function App() {
  return (
    <div>
      <h1>Welcome to bookstore App</h1>
      <BookManager/>
    </div>
  );
}

export default App;
