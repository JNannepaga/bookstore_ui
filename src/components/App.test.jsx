import React from "react";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import App from "./App";

describe("Component - App",()=>{
    it('Matches snapshot',()=>{
        const { asFragment } = render(<App />);
        expect(asFragment()).toMatchSnapshot();
    })
});