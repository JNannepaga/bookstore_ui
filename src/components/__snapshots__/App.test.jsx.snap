// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - App Matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Welcome to bookstore App.
    </h1>
  </div>
</DocumentFragment>
`;
